### PR TITLE
Add Deluge osu!catch difficulty naming `Difficulty_naming`

### PR DESCRIPTION
**Pull request by Darius-Wattimena**
I had this on my TODO list to add to the wiki but never found the time.

People that don't know this difficulty naming see for example the following sets: [here](https://osu.ppy.sh/beatmapsets/1151836#fruits/2456885) and [here](https://osu.ppy.sh/beatmapsets/784412#fruits/1646939).

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by TicClick** [comment]
minor nit: instead of outdating the translations, could you update them? the only difference from the line above yours is a difficulty name, so you can safely copy and paste it

**Comment by Darius-Wattimena** [comment]
Good point, didn't think about that, I don't really speak the language but I can copy-paste stuff for sure

**Review by TicClick — APPROVED** [review]

File: wiki/Ranking_Criteria/Difficulty_naming/id.md [hunk 1]
```diff
@@ -197,3 +197,4 @@ Nama tingkat kesulitan berikut adalah alternatif umum dan tambahan untuk nama ti
 - ![](/wiki/shared/diff/expert-o.png?20211215) Extra: dapat digunakan sebagai pengganti Expert.
 - ![](/wiki/shared/diff/expertplus-o.png?20211215) Extreme: dapat digunakan sebagai Expert yang lebih sulit.
 - ![](/wiki/shared/diff/expertplus-t.png?20211215) Hell Oni: dapat digunakan sebagai Inner Oni yang lebih sulit.
+- ![](/wiki/shared/diff/expertplus-c.png?20211215) Deluge: dapat digunakan sebagai Deluge yang lebih sulit.
```
Comment (Kimitakari) [inline wiki/Ranking_Criteria/Difficulty_naming/id.md:200]: Deluge -> Overdose in explanation

Comment (Darius-Wattimena) [reply wiki/Ranking_Criteria/Difficulty_naming/id.md:200]: Oh, I copied over it for all translations ... let me fix that

**Review by TicClick — APPROVED** [review]
thanks for catching that

**Pull request merged**